Agents/GFAAgent/WEB-INF/web.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://xmlns.jcp.org/xml/ns/javaee" xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd" metadata-complete="false" version="3.1">
+    <display-name>OSMAgent</display-name>
+    <description>A simple java agent for osm data.</description>
+
+    <servlet>
+        <servlet-name>OSMAgent</servlet-name>
+        <servlet-class>uk.ac.cam.cares.jps.agent.osmagent.osmagent</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>OSMAgent</servlet-name>
+        <url-pattern>/</url-pattern>
+    </servlet-mapping>
+
+
+</web-app>
```

Comment:
This file is not needed when the @WebServlet annotation is used in the agent (also they are for OSMAgent). However, the WEB-INF folder is required for the log4j configuration file